### PR TITLE
[Feature #121950151] Create the landing page

### PR DESCRIPTION
**Pull request by monicaah**
### What this PR does

Setups react-router in the landing page.
Adds the entire landing page UI to project

Description of task to be completed
Create the landing page with routes setup
### What are the relevant pivotal tracker stories

[Accepts #121950151]
[Accepts #128491295]

### CODE REVIEW

File: app/components/pages/Contact.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+
+class Contact extends React.Component {
+  render() {
+    return <div> Contact </div>;
```
Comment (BethMwangi) [inline app/components/pages/Contact.js]: ## Code Quality - Nitpick

The return should be enclosed in braces

File: app/components/pages/About.js [hunk 2]
```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+
+class About extends React.Component {
+  render() {
+    return <div> About </div>;
```
Comment (joykare) [inline app/components/pages/About.js]: ## Code Quality - Nitpicks
- The return should be enclosed in braces.
- Class names should be the given fie names in Camel Case.

File: app/components/pages/Contact.js [hunk 3]
```diff
@@ -0,0 +1,9 @@
+import React from 'react';
```
Comment (joykare) [inline app/components/pages/Contact.js]: ## Code Quality - Nitpicks
- React Components should be in files with the extension .jsx

File: app/components/pages/Contact.js [hunk 4]
```diff
@@ -0,0 +1,9 @@
...
+
+class Contact extends React.Component {
```
Comment (joykare) [inline app/components/pages/Contact.js]: ## Code Quality - Nitpicks
- Export classes as you create them. One liners - Neat, clean and less code

File: app/components/pages/LandingPage/appbar.jsx [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+        </ul>
+
+        </nav>
+        {this.props.children || <Home/>}
+      </div>)
+    ;
+  }
+}
+
+App.propTypes = {
```
Comment (joykare) [inline app/components/pages/LandingPage/appbar.jsx]: ## Code Quality - Nitpicks
- PropTypes can be declared within classes.

File: app/index.tpl.html [hunk 6]
```diff
@@ -4,6 +4,8 @@
     <meta charset="utf-8">
     <meta http-equiv="x-ua-compatible" content="ie=edge">
     <title>Panda Analytics</title>
+    <link rel="stylesheet" href="style.scss" />
```
Comment (joykare) [inline app/index.tpl.html]: ## Code Quality
- Remove the scss reference since its not being used.

File: app/style.css [hunk 7]
```diff
@@ -0,0 +1,206 @@
+* {
+  margin: 0;
+  padding: 0;
+  font-family: 'Lato';}
```
Comment (joykare) [inline app/style.css]: ## Code Quality - Nitpicks
- Parenthesis on new line

File: app/style.css [hunk 8]
```diff
@@ -0,0 +1,206 @@
...
+  height: 5vh;
+  margin: auto; }
+
+.social_icons img {
+  width: 20%;
+  margin: 12% 2% 0% 6%; }
+
+footer p {
+  font-size: 1em;
+  color: #666666;
```
Comment (joykare) [inline app/style.css:236]: ## Code Quality - Nitpicks
- Can use three digits instead of six when they are the same.

**Pull request merged**